packages/core/stories/form-field/form-field.stories.tsx

```diff
@@ -184,67 +183,81 @@ export const MultiChild: ComponentStory<typeof FormField> = (props) => {
     <FlowLayout style={{ width: "366px" }}>
       <FormField validationStatus={valid ? undefined : "error"} {...props}>
         <FormLabel>Multi criteria inputs</FormLabel>
-        <FormFieldControlWrapper>
-          <Input
-            value={firstValue}
-            onChange={(e: ChangeEvent<HTMLInputElement>) => {
-              setFirstValue(e.target.value);
-            }}
-          />
-          <Input
-            value={secondValue}
-            onChange={(e: ChangeEvent<HTMLInputElement>) => {
-              setSecondValue(e.target.value);
-            }}
-            placeholder="Multiplier"
-          />
-        </FormFieldControlWrapper>
+        <StackLayout gap={1} direction={"row"}>
```

Comment:
How is this supposed to work when fieldset is the direct child, not the inputs?

Reply:
doh i wasn't thinking 🤕 changed it to role=group